scripts/Units/armvader.bos

```diff
@@ -98,7 +98,8 @@ AimWeapon2()
 
 FireWeapon1()
 {
-	emit-sfx 4097 from leg1; //Weapon2 detonates the crawling bomb once weapon1 fires
+	get KILL_UNIT(get MY_ID,TRUE,FALSE);
+	//emit-sfx 4097 from leg1; //Weapon2 detonates the crawling bomb once weapon1 fires
```

Comment:
Is this line supposed to be uncommented in the scripts? If not, it should be removed.

Reply:
Cleaned up